package.json

```diff
@@ -28,39 +29,57 @@
     "product-type",
     "export"
   ],
+  "files": [
+    "dist",
+    "bin"
+  ],
   "author": "Philipp Sporrer <philipp.sporrer@commercetools.de>",
   "license": "MIT",
   "devDependencies": {
     "babel-cli": "6.6.0",
     "babel-core": "6.6.5",
-    "babel-eslint": "5.0.0",
-    "babel-istanbul": "0.6.1",
+    "babel-eslint": "^6.1.2",
+    "babel-plugin-istanbul": "^2.0.1",
     "babel-preset-es2015": "6.6.0",
     "babel-preset-stage-0": "6.5.0",
-    "chai": "3.5.0",
-    "codecov.io": "0.1.6",
+    "chokidar-cli": "^1.2.0",
```

Comment:
I see you are using this instead of `watch`. Please remember to update the boilerplate repo as well if you use other dependencies